sympy/core/relational.py

```diff
@@ -1075,9 +1087,79 @@ def _n2(a, b):
             return dif
 
 
+@dispatch(Rational, Pow)
+def _eval_is_ge(rat, rhs): # noqa:F811
+    # i >= b**((+/-1)/j)
+    if rat.is_zero:
+        return
+    base, exp = rhs.as_base_exp()
+    if base.is_nonnegative:
+        if is_le(exp, S.NegativeOne):
+            return is_ge(base, rat)
+        if is_le(base, S.One):
+            return is_ge(rat, S.One)
+
+
+@dispatch(Pow, Rational)
+def _eval_is_ge(lhs, rat): # noqa:F811
+    # b**((+/-1)/j) >= i
+    if rat.is_zero:
+        return
+    base, exp = lhs.as_base_exp()
+    if base.is_nonnegative:
+        if is_ge(base, S.One) and is_ge(exp, S.One):
+            return is_le(rat, S.One)
```

Comment:
So we want to know if `a**b >= c`. The inference here is
```
a >= 1  &  b >= 1    --->     a**b >= c  iff  c <= 1
```
That's not an if and only if though because e.g. if a = 2 and b = 2 and c = 2 then this would return False even though the statement is true.